internal/docs/android.md

```diff
@@ -42,3 +42,9 @@ identifier, icon, permissions, ...)
 
 For example: `android:icon="@drawable/icon"` attribute can be added to
 `application` in `AndroidManifest.xml` to use the `icon.png`.
+
+## Java native interface
+
+You can call android's java runtime directly from go. See [java native interface example](internal/examples/androidextras/jni)
```

Comment:
please replace `(internal/examples/androidextras/jni)` with `(../examples/androidextras/jni)`.

current link leads to a 404 page.